osu.Game/Overlays/UserProfileOverlay.cs

```diff
@@ -40,9 +40,7 @@ public void ShowUser(User user, bool fetchOnline = true)
             if (user.Id == Header?.User.Value?.Id)
                 return;
 
-            userReq?.Cancel();
```

Comment:
This changes the logic. It now makes it possible for an old request to complete with no drawable content, or with content that should be used to display the new user.

Reply:
Rewritten to avoid ambiguity.